bigbluebutton-html5/imports/api/video/server/handlers/userSharedHtml5Webcam.js

```diff
@@ -3,17 +3,18 @@ import { check } from 'meteor/check';
 
 export default function handleUserSharedHtml5Webcam({ header, body }, meetingId ) {
   const { userId, stream } = body;
-  const isValidStream = Match.Where((stream) => {
-    check(stream, String);
+  const isValidStream = (match) => {
```

Comment:
Please consider changing `match` with `testString` for both `handleUserSharedHtml5Webcam` and `handleUserUnsharedHtml5Webcam`. Otherwise all looks good!